framework/src/base/NonlinearSystem.C

```diff
@@ -870,6 +870,7 @@ NonlinearSystem::enforceNodalConstraintsJacobian(SparseMatrix<Number> & jacobian
       }
     }
   }
+  jacobian.close();
```

Comment:
Will this still work if multiple NodalConstraints add to the same jacobian entries before the close() is called?